### ISSUE TITLE
Crawling through proxy while using normal network for YaCy to YaCy communication

### ISSUE THREAD

**Issue by Orangian** [yacy/yacy_search_server#410]
Hi. I’m using privoxy as my reverse proxy routed through tor to crawl websites as I don’t want to get my actual network blocked by them (even temporarily), however, I have not found a way to have my server accessible by others. I don’t care about having it exposed to other peers so is there any way to tell it to use normal networking for YaCy to YaCy communication, but a proxy only for web crawling?

It specifically says this on the status page:
```
Proxy
Transparent off   URL off
Remote: 127.0.0.1:8118 | Used for YaCy -> YaCy communication: Yes
```
So it seems like there might be a way to disable YaCy to YaCy communication through the proxy but keep crawling on?

**Comment by Tarcaxoxide** [comment]
I noticed this 2.
I actually just want yacy to pass along connections using it's proxy to the privoxy proxy but everything else should operate normally.

like if i do `http://yourpeer:yourport/proxy.html?url=http://externalurl/ ` than it should pass this request to privoxy.
nothing else should use privoxy.

**Comment by Orangian** [comment]
Yeah, this would help immensely. I know a lot of things block requests originating from VPN's or the Tor network, but I don't want my personal network to get blocked or marked as a crawler.
